integration-tests/src/test/resources/xpack_cluster_with_ror_nodes_and_internode_ssl/readonlyrest.yml

```diff
@@ -1,11 +1,9 @@
 readonlyrest:
   ssl:
     enable: true
-    keystore_file: "elastic-certificates.p12"
-    keystore_pass: readonlyrest
-    truststore_file: "elastic-certificates.p12"
-    truststore_pass: readonlyrest
-    key_pass: readonlyrest
+    server_certificate_key_file: elastic-certificates-pkey.pem
```

Comment:
maybe we should run XpackClusterWithRorNodesAndInternodeSslSuite twice? I mean, it'd be good to test it with p12 and pem. So, maybe we can have two configs (for modules > es60x obviously) and reuse the suite to create two plugin tests? WDYT?

Reply:
sounds good. I wanted to test both functionalities by using pem files for server and keystore for internode, but this solution seems better